apps/workers/sql-load/worker.go

```diff
@@ -96,6 +111,11 @@ func (o *options) newWorker(info string) task.Worker {
 	if w.Params.Truncate {
 		w.delQuery = fmt.Sprintf("delete from %s", w.Params.Table)
 	}
+
+	if len(w.delQuery) > 0 {
+		log.Println("delete query found", w.delQuery)
+	}
```

Comment:
Do we need this? It seems to be only for logging.

Reply:
I can remove it. It was just a "nice to know" kind of thing.